src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/multi-cluster/multi-cluster.component.ts

```diff
@@ -170,6 +175,24 @@ export class MultiClusterComponent implements OnInit {
     this.bsModalRef = this.modalService.show(MultiClusterFormComponent, initialState, {
       size: 'lg'
     });
+    this.bsModalRef.componentInstance.submitAction.subscribe(() => {
+      this.loading = true;
+      setTimeout(() => {
+        this.multiClusterService.refresh();
+        this.summaryService.refresh();
+        const currentRoute = this.router.url.split('?')[0];
+        if (currentRoute.includes('dashboard')) {
+          this.router.navigateByUrl('/pool', { skipLocationChange: true }).then(() => {
+            this.router.navigate([currentRoute]);
+          });
+        } else {
+          this.router.navigateByUrl('/', { skipLocationChange: true }).then(() => {
+            this.router.navigate([currentRoute]);
+          });
+        }
+        this.getPrometheusData(this.prometheusService.lastHourDateObject);
+      }, 20000)
```

Comment:
The interval here could be set with a constant value, for better clarity

Reply:
Done. Thanks @Pegonzal